pkg/config/analysis/msg/messages.yaml

```diff
@@ -601,3 +601,18 @@ messages:
         type: int
       - name: selector
         type: string
+
+  - name: "MultiClusterInconsistentService"
+    code: IST0163
+    level: Warning
+    description: "The services lived in different clusters under multi-cluster deployment model are inconsistent"
```

Comment:
```suggestion
    description: "The services live in different clusters under multi-cluster deployment model are inconsistent"
```

Reply:
would be good to clarify how inconsistent is defined -- via service name or its namespace?

Reply:
Thanks